docs/reference/crud.txt

```diff
@@ -228,56 +410,56 @@ operations with examples.
           person.touch
           person.touch(:audited_at)
 
-   * - ``Model#delete``
+   * - ``Model#destroy``
```

Comment:
what was the purpose of switching the order of delete and destroy here? 

I think it was put in this order for the sake of alphabetical sorting...

Reply:
same for delete_all, destroy_all

Reply:
I personally find a "logical" order easier to read and follow. With that said, I can see how what is "logical" can vary from one person to the next.

In this case, `destroy` is an operation which should be used by default to remove documents, and `delete` should be explicitly chosen over `destroy` for performance reasons after considering whether there would be orphan objects left, whether any callbacks would be skipped, etc. Thus `destroy` comes first and `delete` comes second in my mind.